### PR TITLE
Tagging-UPAVAN Bugs

### PR DESCRIPTION
**Pull request by djangobotpython**

### CODE REVIEW

File: dg/media/coco/app/scripts/views/form.js [hunk 1]
```diff
@@ -988,14 +991,14 @@ define([
...
-                            $('.search-choice-close').click();
+                            $(that.entity_config.reset_element+'_chosen'+ ' '+ '.search-choice-close').click();
                             $(that.entity_config.reset_element).trigger("chosen:updated");
                         }
                     })
                 }
                 if (this.edit_case && this.foreign_elements_rendered[element]){
                     $("#id_"+ this.entity_config.fetch_element_that_manipulate + "_chosen").on('click', function(){
-                        $('.search-choice-close').click();
+                        $(that.entity_config.reset_element+'_chosen'+ ' '+ '.search-choice-close').click()
```
Comment (Abhishek-lodha) [inline dg/media/coco/app/scripts/views/form.js]: put a semi colon

File: dg/media/dashboard.html [hunk 2]
```diff
@@ -19,7 +19,7 @@
     <LINK REL="stylesheet"  HREF="/media/coco/app/styles/css/tabletools_custom.css">
     <LINK rel="icon" type="image/x-icon" href="/media/coco/app/images/favicon.ico">
 
-    <script data-main="/media/coco/dist/scripts/main" src="/media/coco/app/scripts/libs/require.js"></script>
+    <script data-main="/media/coco/app/scripts/main" src="/media/coco/app/scripts/libs/require.js"></script>
```
Comment (Abhishek-lodha) [inline dg/media/dashboard.html]: why is directory changed from dist to app

Comment (djangobotpython) [reply dg/media/dashboard.html]: Explained to you.

File: dg/media/coco/app/scripts/views/form.js (hunk 1 above)
Comment (djangobotpython) [reply dg/media/coco/app/scripts/views/form.js]: Okay

**Pull request merged**